test/set.js

```diff
@@ -126,7 +126,7 @@ var thrift = new Thrift({source: source});
 test('Struct with set rw', testRW.cases(thrift.Bucket.rw, [
 
     [new thrift.Bucket({asArray: [1, 2, 3]}), [
-        0x0f,                   // type:1      -- 15, struct
+        0x0e,                   // type:1      -- 14, set
```

Comment:
It is unclear to me how this ever worked with the struct typeid. I expected it would have been list.